tests/api/Api/CleanInputCest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Test\Api\Api;
```

Comment:
Api\Api, that's confusing

Reply:
When all endpoints are moved, we can move this up and remove the second in the namespace (would be messy now)